guides/database-integrating-with-postgresql.adoc

```diff
@@ -0,0 +1,739 @@
+=== PostgreSQL
+
+We will use PostgreSQL as the database server in its containerized version: see https://hub.docker.com/_/postgres[PostgreSQL Official Image, window=_blank].
+
+Start PostgreSQL database in a container with:
+
+[source,bash,subs="normal"]
+----
+podman run --rm --name bookstore \
+  -p {postgre-sql-port}:{postgre-sql-port} \
+  -e POSTGRES_PASSWORD={postgre-sql-password} \
+  -e POSTGRES_USER={postgre-sql-user} \
+  -e POSTGRES_DB={postgre-sql-database} \
+  {postgre-docker-image}
+----
+
+NOTE: we started the container with the `--rm` flag so it can be disposed of automatically when we stop it.
+
+
+== Application
+
+=== Create a new Maven project
+
+We are going to use the *WildFly Getting Started Archetype* to create the base structure of our Book Store API application.
+
+Open a new terminal window and create a new project using the WildFly Getting Started Archetype:
+
+[source,bash,subs="normal"]
+----
+mvn archetype:generate \
+    -DarchetypeGroupId=org.wildfly.archetype \
+    -DarchetypeArtifactId=wildfly-getting-started-archetype \
+    -DdefaultClassPrefix=BookStore \
+    -DartifactId=bookstore \
+    -Dversion=1.0.0 \
+    -DinteractiveMode=false
+----
+Remove the following files from the base project since we are not going to use them:
+
+[source,bash]
+----
+cd bookstore
+rm src/main/java/org/wildfly/examples/BookStoreService.java
+rm src/main/java/org/wildfly/examples/BookStoreEndpoint.java
+rm src/test/java/org/wildfly/examples/BookStoreApplicationIT.java
+rm src/test/java/org/wildfly/examples/BookStoreServiceIT.java
+----
+
+=== pom.xml
+
+==== Jakarta EE dependencies:
+
+Add the following dependencies to the `pom.xml` file:
+
+[source,xml]
+----
+<dependency>
+    <groupId>jakarta.persistence</groupId>
+    <artifactId>jakarta.persistence-api</artifactId>
+    <scope>provided</scope>
+</dependency>
+<dependency>
+    <groupId>jakarta.transaction</groupId>
+    <artifactId>jakarta.transaction-api</artifactId>
+    <scope>provided</scope>
+</dependency>
+<dependency>
+    <groupId>jakarta.validation</groupId>
+    <artifactId>jakarta.validation-api</artifactId>
+    <scope>provided</scope>
+</dependency>
+----
+
+==== Configure WildFly Datasource and trimming server capabilities:
+To connect to the database we need to configure the https://docs.wildfly.org/32/Admin_Guide.html#DataSource[WildFly Datasource Subsystem,window=_blank] and install the PostgreSQL driver into the WildFly server. The https://github.com/wildfly-extras/wildfly-datasources-galleon-pack[WildFly Datasource Galleon Pack, window=_blank] contains a set of https://github.com/wildfly-extras/wildfly-datasources-galleon-pack/blob/main/doc/postgresql/README.md[Galleon Layers, window=_blank] that provide *JDBC drivers* and *WildFly Datasource Subsystem* configurations for various databases. For this guide, we will use the `postgresql-default-datasource` Galleon layer that will configure a PostgreSQL datasource as the default datasource for the server.
```

Comment:
s/WildFly Datasource Galleon Pack/WildFly Datasources Galleon Pack/g

Reply:
Done.

Replaced all the occurrences.